.github/workflows/release-gh-notes.yml

```diff
@@ -0,0 +1,53 @@
+name: release-gh-notes
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        description: "Specify the version for this release"
+        type: string
+      npm_package:
+        required: true
+        description: "npm package of the release"
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: install with retry # Retry 5 times with 1 minute delay to guarantee that the version is released to npm
```

Comment:
We already have some npm integration tests which are called right after every publish and they never failed. Can we check how much necessary this is?

Reply:
I have checked and it seems not necessary, we can always add it back if it doesn't work